src/main/scala/com/datastax/gatling/plugin/model/DseCqlAttributesBuilder.scala

```diff
@@ -134,20 +153,9 @@ case class DseCqlAttributesBuilder(attr: DseCqlAttributes) {
     * @return
     */
   @deprecated("Replaced by withConsistencyLevel")
-  def consistencyLevel(level: ConsistencyLevel) = withConsistencyLevel(level)
-
-
-  /**
-    * For Backwards compatibility
-    *
-    * @see [[DseCqlAttributesBuilder.executeAs]]
-    * @param userOrRole User or role to use
-    * @return
-    */
-  @deprecated("Replaced by withUserOrRole")
-  def executeAs(userOrRole: String) = withUserOrRole(userOrRole: String)
-
-  def check(check: DseCqlCheck) = DseCqlAttributesBuilder(attr.copy(cqlChecks = check :: attr.cqlChecks))
+  def consistencyLevel(level: ConsistencyLevel):DseCqlAttributesBuilder[T, B] =
+    withConsistencyLevel(level)
 
-  def check(check: GenericCheck) = DseCqlAttributesBuilder(attr.copy(genericChecks = check :: attr.genericChecks))
+  def check(check: DseCqlCheck):DseCqlAttributesBuilder[T, B] =
+    DseCqlAttributesBuilder(attr.copy(cqlChecks = (attr.cqlChecks :+ check)))
```

Comment:
A subtle API change that becomes more significant as we focus to checks based on AsyncResultSet directly; checks should always be evaluated in the order in which they are declared.  This allows you to do routine things like check the output of remaining() and/or hasMorePages() before doing things that might impact the return values of these functions... like, say, getting a row from the ARS.  I was seeing remaining() checks followed by a row contents check fail because the old order eval'd the checks in reverse order... so the one() call happened before the remaining check, which threw off the count.